src/table.rs

```diff
@@ -63,23 +120,56 @@ enum CheckStatus {
     Fail,
 }
 
-pub fn print_tables(json_struct: Value, options: &Options) -> anyhow::Result<ExitCode> {
+pub fn adjust_color(level: String, colors: &Colored) -> Cell {
+    return match level.to_lowercase().as_str() {
+        "info" => Cell::new(format!("{level:?}")).fg(colors.severity_info),
+        "low" => Cell::new(format!("{level:?}")).fg(colors.severity_low),
+        "medium" => Cell::new(format!("{level:?}")).fg(colors.severity_medium),
+        "high" => Cell::new(format!("{level:?}")).fg(colors.severity_high),
+        "high" => Cell::new(format!("{level:?}")).fg(colors.severity_high),
+        &_ => Cell::new(format!("{level:?}")).fg(colors.text),
+    };
+}
+
+pub fn print_tables(
+    json_struct: Value,
+    options: &Options,
+    use_colors: bool,
+) -> anyhow::Result<ExitCode> {
+    let color_choice = if !use_colors {
```

Comment:
This whole thing with the ColorChoice seems rather redundant, you should have just used the static vars directly without an enum

Reply:
Yes , I understand so I removed the enum.